csrc/common.h

```diff
@@ -8,7 +8,7 @@ using namespace BinSearch;
 #define BLOCK_SIZE 16384
 
 struct quantize_block_args {
-    BinAlgo<Scalar, float, Direct2> *bin_searcher;
+    BinAlgo<AVX, float, Direct2> *bin_searcher;
```

Comment:
Does not compile with Scalar. TBH I have no idea how this works with Linux compilation, there are missing template definitions.